### PR TITLE
BRP-95: Add case ID to not-arrived letter-received route and reformat caseworker emails

### PR DESCRIPTION
**Pull request by SarahJaneLuff**
**Changes**

- letter-received.html updated to include new case-id-number fields in not-arrived route.
- new case-id-number fields added to pages.json, fields.json, and index.js within not-arrived route.
- Email templates changed to include a comma after the field name, and updated to include all fields in the email with 'None Given' for fields with no information obtained from the user.
- Updated acceptance tests to include the new case-id-number field.
- Fixed data issues with email, phone number and county address not appearing on emails on all routes.

**Reason**
Changes above requested by the business.

### CODE REVIEW

File: apps/not-arrived/translations/src/en/fields.json [hunk 1]
```diff
@@ -47,6 +47,10 @@
       }
     }
   },
+  "case-id-number": {
+    "label": "Case ID Number",
+    "hint": "(Enter the case ID found on your visa decision letter.)"
```
Comment (sulthan-ahmed) [inline apps/not-arrived/translations/src/en/fields.json:52]: Should it be capital for case?

File: services/email/templates/caseworker/html/someone-else.mus [hunk 2]
```diff
@@ -122,6 +120,8 @@
                         </td>
                       </tr>
                     {{/someone-else-reason-radio}}
+
+
```
Comment (sulthan-ahmed) [inline services/email/templates/caseworker/html/someone-else.mus:124]: Is the a reason for two extra lines?

File: apps/not-arrived/translations/src/en/fields.json (hunk 1 above)
Comment (SarahJaneLuff) [reply apps/not-arrived/translations/src/en/fields.json:52]: They use the same wording else where in the service and its a lower case:

"case-id": {
      "label": "Case ID number",
      "hint": "(Enter the case ID found on your visa decision letter.)"
},

File: services/email/templates/caseworker/html/someone-else.mus (hunk 2 above)
Comment (SarahJaneLuff) [reply services/email/templates/caseworker/html/someone-else.mus:124]: removed the extra line

**Pull request merged**